mapper/orm/src/main/java/org/hibernate/search/mapper/orm/search/loading/impl/HibernateOrmByTypeEntityLoader.java

```diff
@@ -47,7 +48,12 @@
 				referencesByDelegate.entrySet() ) {
 			HibernateOrmComposableEntityLoader<? extends T> delegate = entry.getKey();
 			List<EntityReference> referencesForDelegate = entry.getValue();
-			delegate.loadBlocking( referencesForDelegate, objectsByReference );
+
+			delegate.loadBlocking( referencesForDelegate, objectsByReference, timeoutManager );
+		}
+
+		if ( timeoutManager != null ) {
+			timeoutManager.stop();
```

Comment:
I believe this should be called by whoever called `start`, not by this class.

Reply:
Good catch. Right, the timeout lifecycle is handled by the backends.